projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -1043,7 +1043,7 @@ public static function get_pending_cached_wpcom_invite( $email ) {
 		 * @static
 		 * @param int $user_id The user ID.
 		 *
-		 * @return {false|string} returns the user invite code if the user is invited, false otherwise.
+		 * @return false|string returns the user invite code if the user is invited, false otherwise.
```

Comment:
Not a blocker, but perhaps we should swap this?
```suggestion
		 * @return string|false returns the user invite code if the user is invited, false otherwise.
```